config/default_config.yml

```diff
@@ -116,35 +116,32 @@ GTX:
 
 
 NOLEADER_AUTO_MOD:
-  # Set to true to enable the auto moderation of squads without officers
-  # This auto-moderator has 3 steps, 1. warning via broadcast (N times), 2. Punish (N times) 3. kick
+  # This auto-moderator has 3 steps, 1. warning via direct message (X times), 2. punish (Y times) 3. kick
   # Note that the state cycle for a given squad only resets once it finally has an officer. So for example if you
   # disabled the kick and have 3 punishes configured, once the 3 punishes are 
   # through that's it, nothing will happen anymore for that squad even if they still dont have an officer. 
   # 
   # Note 2: punishes and kicks are tracked individual per player, not at the squad level.
   # So if a player joins a squad that already received 3 punishes and will be kicked next, the new player will 
-  # start at the first punish regardless (so he won't be kick), howver he won't have a warning broadcast message as those are issued and tracked per squad)
-  enabled: false
+  # start at the first punish regardless (so he won't be kick)
+  # Set to true to enable the auto moderation of squads without officers
+  enabled: true
```

Comment:
hmm I'd not change that default just yet

Reply:
Yes good catch, residue of testing